src/vs/workbench/parts/extensions/electron-browser/extensionTipsService.ts

```diff
@@ -381,6 +382,16 @@ export class ExtensionTipsService extends Disposable implements IExtensionTipsSe
 		return product.keymapExtensionTips || [];
 	}
 
+	getRestoredRecommendationsWithReason(reasonId: ExtensionRecommendationReason): string[] {
```

Comment:
The `withReason` in the name here conflicts with the `withReason` in `getRecomemndationsWithReason`
Rename either.